Small improvements

jooweg opened the pull request:
- Added taskname input option
    - Better sequence checking
    - Incomplete sequence warning made less alarming
    - AAScout protocol added (is not processed)

jooweg committed:
Add files via upload

    - Added taskname input option
    - Better sequence checking
    - Incomplete sequence warning made less alarming
    - AAScout protocol added (is not processed)

dangom commented on dac2bids.py:
```diff
@@ -293,8 +303,12 @@ def main():
     p.add_option('--skipfmap', '-f', action="store_true", help="Skips fieldmaps.")
+    p.add_option('--taskname', '-t', default="task-unknown", help="The task name")    
 
+    global options 	# make options globally available
```
Global is usually a bad idea because it allows `options` to be mutated from any place in the code. This can lead to subtle bugs. Also, if we were to import dac2bids and not call it as a main, then `options` would be used within `parse_protocols` without ever having been defined, leading to a `NameError`.

dangom commented on dac2bids.py:
```diff
@@ -293,8 +303,12 @@ def main():
+    global options 	# make options globally available
     options, arguments = p.parse_args()
+    if not 'task-' in options.taskname:
+        options.taskname = 'task-'+options.taskname
```
A better solution to the `global` issue, also for consistency with the rest of the code (the handling of subjects and sessions), is to add the taskname option as an extra keyword argument for `create_yaml`.

jooweg committed:
removed global variables

- Added taskname option as an extra keyword argument for create_yaml and parse_protocols

jooweg commented:
Yes, that is a way better solution. I've changed it.
